src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -3427,6 +3427,28 @@ void clang_c_convertert::get_decl_name(
     }
     break;
 
+  case clang::Decl::CXXConstructor:
```

Comment:
I'm wondering whether this case wouldn't be better handled in the clang-cpp-frontend instead of the C one, seeing that it is an entire new case block that could easily be moved (as opposed to your previous PR, which just handles the two C and C++ nodes with the same identical code path).

Reply:
Up to you I guess.

Reply:
Yes, please, if it's not too much trouble. :)

Reply:
Done.